purchases/src/test/java/com/revenuecat/purchases/models/PeriodTest.kt

```diff
@@ -52,6 +52,22 @@ class PeriodTest {
         assertThat(period.iso8601).isEqualTo("P3M")
     }
 
+    @Test
+    fun `create period creates expected period class for multiple different units`() {
+        val period = Period.create("P12W6D")
```

Comment:
Nice! Does this mean we support both ISO 8601-1 and ISO 8601-2 now? If so, maybe we can specify that in the documentation of the `Period.create()` function? (Is that meant to be used by developers?)

Reply:
Yup! It should support both. And yeah, currently that doesn't have documentation, I think the only reason to keep it public is to help with devs creating instances for tests but shouldn't really be used in a production environment... So I will add some notes about that. Good catch!